Classes/Infrastructure/ContentRepository/ConflictsFactory.php

```diff
@@ -130,67 +124,42 @@ private function createConflictFromCommandThatFailedDuringRebase(
-    {
-        return match (true) {
-            $command instanceof MoveNodeAggregate,
-            $command instanceof SetNodeProperties,
-            $command instanceof SetSerializedNodeProperties,
-            $command instanceof CreateNodeAggregateWithNode,
-            $command instanceof CreateNodeAggregateWithNodeAndSerializedProperties,
-            $command instanceof TagSubtree,
-            $command instanceof DisableNodeAggregate,
-            $command instanceof UntagSubtree,
-            $command instanceof EnableNodeAggregate,
-            $command instanceof RemoveNodeAggregate,
-            $command instanceof ChangeNodeAggregateType,
-            $command instanceof CreateNodeVariant =>
-                $command->nodeAggregateId,
-            $command instanceof SetNodeReferences,
-            $command instanceof SetSerializedNodeReferences =>
-                $command->sourceNodeAggregateId,
-            default => null
-        };
-    }
-
-    private function acquireSubgraphFromCommand(
-        CommandInterface $command,
+    private function acquireSubgraph(
+        EventInterface $event,
         ?NodeAggregateId $nodeAggregateIdForDimensionFallback
     ): ?ContentSubgraphInterface {
         if ($this->workspace === null) {
             return null;
         }
 
-        $dimensionSpacePoint = match (true) {
-            $command instanceof MoveNodeAggregate =>
-                $command->dimensionSpacePoint,
-            $command instanceof SetNodeProperties,
-            $command instanceof SetSerializedNodeProperties,
-            $command instanceof CreateNodeAggregateWithNode,
-            $command instanceof CreateNodeAggregateWithNodeAndSerializedProperties =>
-                $command->originDimensionSpacePoint->toDimensionSpacePoint(),
-            $command instanceof SetNodeReferences,
-            $command instanceof SetSerializedNodeReferences =>
-                $command->sourceOriginDimensionSpacePoint->toDimensionSpacePoint(),
-            $command instanceof TagSubtree,
-            $command instanceof DisableNodeAggregate,
-            $command instanceof UntagSubtree,
-            $command instanceof EnableNodeAggregate,
-            $command instanceof RemoveNodeAggregate =>
-                $command->coveredDimensionSpacePoint,
-            $command instanceof ChangeNodeAggregateType =>
+        $dimensionSpacePoint = match ($event::class) {
+            NodeAggregateWasMoved::class =>
+                $event->succeedingSiblingsForCoverage->toDimensionSpacePointSet()->getFirst(),
+            NodePropertiesWereSet::class,
+            NodeAggregateWithNodeWasCreated::class =>
+                $event->originDimensionSpacePoint->toDimensionSpacePoint(),
+            NodeReferencesWereSet::class =>
+                $event->affectedSourceOriginDimensionSpacePoints->toDimensionSpacePointSet()->getFirst(),
```

Comment:
not ideal that there is only a set here and not a `originDimensionSpacePoint` like in `SetSerializedNodeProperties`?